starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -16,103 +16,9 @@ else
-relevant_mount_point_arguments["--output-directory"]=out_path
-relevant_mount_point_arguments["--temporary-base-directory"]=out_path
-relevant_mount_point_arguments["--cache-directory"]=in_path
-relevant_mount_point_arguments["--save-directory"]=out_path
-relevant_mount_point_arguments["--task-dependencies-dot-file"]=out_file
-#relevant_mount_point_arguments["--test-folder"]=in_path # TODO reactive in #37
-#relevant_mount_point_arguments["--test-file"]=in_file # TODO reactive in #37
-
-function _get_mount_point_path_for_in_dir() {
-  local current_arg=$1
-  local dir_path=$2
-
-  if [[ -d $dir_path ]]; then
-    mount_point_paths+=("$dir_path")
-  else
-    echo "Input directory $dir_path for parameter $current_arg does not exist."
-    exit 1
-  fi
-}
-
-function _get_mount_point_path_for_in_file() {
-  local current_arg=$1
-  local file_path=$2
-  if [[ -f $file_path ]]; then
-    local rel_dir_name=''
-    rel_dir_name="$(dirname "${file_path}")"
-    mount_point_paths+=("$rel_dir_name")
-  else
-    echo "Input file $file_path for parameter $current_arg does not exist."
-    exit 1
-  fi
-}
-
-function _get_mount_point_path_for_out_path() {
-  local dir_path=$1
-  #Create out directories if necessary
-  if [[ ! -d $dir_path ]]; then
-    mkdir -p "$dir_path"
-  fi
-  mount_point_paths+=("$dir_path")
-}
-
-function _get_mount_point_path_for_out_file() {
-  local file_path=$1
-  local rel_dir_name=''
-  rel_dir_name="$(dirname "${file_path}")"
-  #Create out directories if necessary
-  if [[ ! -d $file_path ]]; then
-    mkdir -p "$rel_dir_name"
-  fi
-  mount_point_paths+=("$rel_dir_name")
-}
-
-function _get_mount_point_path() {
-  local current_arg=$1
-  local next_arg=$2
-  local arg_type=$3
-
-  case $arg_type in
-  in_path)
-    _get_mount_point_path_for_in_dir "${current_arg}" "${next_arg}"
-    ;;
-  in_file)
-    _get_mount_point_path_for_in_file "${current_arg}" "${next_arg}"
-    ;;
-  out_path)
-    _get_mount_point_path_for_out_path "${next_arg}"
-    ;;
-  out_file)
-    _get_mount_point_path_for_out_file "${next_arg}"
-    ;;
-  *)
-    echo "INVALID ARGUMENT. Please adjust variable relevant_mount_point_arguments in $0!"
-    exit 1
-    ;;
-  esac
-}
-
-function _get_mount_point_paths() {
-  local lenArgs="$#"
-  for ((idxArg = 1; idxArg < lenArgs; idxArg++)); do
-    current_arg=${!idxArg}
-    next_arg_idx=$((idxArg + 1))
-    next_arg=${!next_arg_idx}
-    #Better way to test if the argument is in $relevant_mount_point_arguments
-    #would be to use [[ -v ...]], however this does not work correctly with bash 4.2
-    if [[ -n "${relevant_mount_point_arguments[${current_arg}]-}" ]]; then
-      _get_mount_point_path $current_arg $next_arg "${relevant_mount_point_arguments[${current_arg}]}"
-    fi
-  done
-}
-
+SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 declare -a mount_point_paths
-_get_mount_point_paths "${@}"
+mount_point_paths=($("$SCRIPT_DIR"/mount_point_parsing.sh "${@}"))
```

Comment:
I guess the problem, why the following test fails is, because we call the mount_point_parsing.sh directly. In the other scripts, I called via bash, that's why they don't fail. We have two options, either call the script via bash or set in the installer the permissions (execute) for the script correctly.

https://github.com/exasol/script-languages-container-tool/pull/59/checks?check_run_id=3118278886#step:3:665